glusterd2/middleware/auth.go

```diff
@@ -79,8 +81,12 @@ func Auth(next http.Handler) http.Handler {
 		})
 
 		// Check if token is Valid
-		if err != nil || !token.Valid {
-			http.Error(w, err.Error(), http.StatusUnauthorized)
+		if err != nil {
+			restutils.SendHTTPError(ctx, w, http.StatusUnauthorized, err.Error())
+			return
+		}
+		if !token.Valid {
+			restutils.SendHTTPError(ctx, w, http.StatusUnauthorized, errors.New("Invalid Token in Authorization header"))
```

Comment:
`token` (all small caps)

`Invalid token specified in Authorization header`

Also, for easy understand-ability, use single quotes around `Authorization` to make it clear that it's a header field. Here and elsewhere

Reply:
done